readme.md

```diff
@@ -181,6 +181,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [lstags](https://github.com/ivanilves/lstags) - Synchronize Docker images across different registries.
 - [pm2](https://pm2.io/runtime/) - Production Process Manager for Node.js.
 - [dockly](https://github.com/lirantal/dockly) - Interactively manage Docker containers.
+- [lazydocker](https://github.com/jesseduffield/lazydocker) - A simple terminal UI for both docker and docker-compose.
```

Comment:
[As stated in the contribution guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md), please move your addition to the end of the category.